articles/spring-cloud/how-to-configure-enterprise-spring-cloud-gateway.md

```diff
@@ -0,0 +1,168 @@
+---
+title: How to Configure Spring Cloud Gateway for Kubernetes with Azure Spring Apps Enterprise Tier
+titleSuffix: Azure Spring Apps Enterprise Tier
+description: How to Configure Spring Cloud Gateway for Kubernetes with Azure Spring Apps Enterprise Tier.
+author: karlerickson
+ms.author: xiading
+ms.service: spring-cloud
+ms.topic: how-to
+ms.date: 02/09/2022
+ms.custom: devx-track-java, devx-track-azurecli, event-tier1-build-2022
+---
+
+# Configure Spring Cloud Gateway for Kubernetes
```

Comment:
@maly7 The phrase "Spring Cloud Gateway for Kubernetes" is confusing for PaaS audience. We tell them that K8S is fully abstracted away from Azure Spring Apps users. Shall we use "Tanzu Spring Cloud Gateway" or something simpler?